### ISSUE TITLE
This foundationdb-arm64 project can  be compiled directly on the arm64 linux machine?

### ISSUE THREAD

**Issue by jianzhenqin** [wolfspider/FoundationDB-ARM64#1]
I see that your README.md is based on linux docker, can it be compiled directly on the arm64 linux machine? The current test environment is chinese kylin system, thanks

**Comment by wolfspider** [comment]
Yes, actually that is how I built it originally. That README was brought
over from when it was forked and I’d say the ARM port is mostly complete
for that version of FDB except for the client side bindings so the build
will error out there but will produce a working server. The bindings for
the client would require some ARM code written in ASM in order for those to
work. Way back when I created this I was given an ARM Chromebook as a loner
and decided to see how far I could get before giving it back to my
coworker. I used Debian Xenial via Crouton. Best of luck! If you get
something working feel free to reach out if you wish or if you have any
more questions let me know.

**Comment by jianzhenqin** [comment]
thank you very much，In README.md (2. Check out the foundationdb repo) refers to foundationdb binary package? Can you describe it in detail, thank you，Because it’s the first time I touched the arm64 platform

**Comment by wolfspider** [comment]
What that would mean is to just clone this repo, really all you need to do
is install the dependencies and then clone this repo and then run the build
scripts. Getting all of Mono installed can be a bit of an issue depending
on how support is with your distro but everything else should install just
fine. "Check out foundationdb repo" should just be read as "clone this
repo".

**Comment by zhupenghui176** [comment]
thanks ,I also encountered this problem, it was perfect，love you!
